backend/src/main/java/com/soen341/instagram/exception/account/UsernameTakenException.java

```diff
@@ -0,0 +1,13 @@
+package com.soen341.instagram.exception.account;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST)
+public class UsernameTakenException extends RuntimeException
+{
+	public UsernameTakenException()
+	{
+		super("Username is taken");
```

Comment:
Just a nice to have: "Username is _already_ taken"?

Reply:
Sure will do